<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Daniel L. Contreras Pino</author>
 <class>frm_GestorNotas</class>
 <widget class="QWidget" name="frm_GestorNotas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="layoutcentral">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QScrollArea" name="frame_notas">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="widget_notas">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>407</width>
          <height>59</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="layout_notas">
         <item>
          <spacer name="hSpacer_notas">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lne_promedio">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="btn_agregarnota">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Agrega una nueva calificaci√≥n.</string>
     </property>
     <property name="text">
      <string notr="true">+</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_agregarnota</sender>
   <signal>clicked(bool)</signal>
   <receiver>frm_GestorNotas</receiver>
   <slot>Agregar_Nota()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Agregar_Nota()</slot>
 </slots>
</ui>
